dist.sh

```diff
@@ -45,9 +45,10 @@ done
 
 rnd=$(LC_ALL=C tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c10)
 docker buildx create --use --name nsq-$rnd
-docker buildx build --tag nsqio/nsq:v$version . --platform linux/amd64,linux/arm64 --push
+docker buildx build --tag nsqio/nsq:v$version --platform linux/amd64,linux/arm64 .
 if [[ ! $version == *"-"* ]]; then
-    echo "Tagging nsqio/nsq:v$version as the latest release."
-    docker buildx build --tag nsqio/nsq:latest . --platform linux/amd64,linux/arm64 --push
+    echo "Tagging nsqio/nsq:v$version as the latest release"
+    shas=$(docker manifest inspect nsqio/nsq:$version |\
+        grep digest | awk '{print $2}' | sed 's/[",]//g' | sed 's/^/nsqio\/nsq@/')
```

Comment:
this looks like a job for `jq` - but if it works for you it's fine with me

Reply:
yes, of course. I wasn't quite sure what our policy for shell compatibility has been, although I suppose you already need quite a few dependencies (golang, docker).